<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FakioWindow</class>
 <widget class="QMainWindow" name="FakioWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>250</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>250</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Fakio 客户端</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Fakio.qrc">
    <normaloff>:/fakio/resources/fakio.ico</normaloff>:/fakio/resources/fakio.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabMain">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>5</y>
      <width>445</width>
      <height>195</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <widget class="QWidget" name="tabStatus">
     <attribute name="title">
      <string>状态</string>
     </attribute>
     <widget class="QPlainTextEdit" name="plainStatus">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>2</y>
        <width>434</width>
        <height>160</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Courier New</family>
       </font>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabSettings">
     <attribute name="title">
      <string>设置</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>105</y>
        <width>31</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>密码</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="passwordEdit">
      <property name="geometry">
       <rect>
        <x>97</x>
        <y>105</y>
        <width>150</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="fakioEdit">
      <property name="geometry">
       <rect>
        <x>97</x>
        <y>11</y>
        <width>321</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="localLabel">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>40</y>
        <width>71</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>本地地址</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="localEdit">
      <property name="geometry">
       <rect>
        <x>97</x>
        <y>40</y>
        <width>321</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="usernameEdit">
      <property name="geometry">
       <rect>
        <x>97</x>
        <y>69</y>
        <width>150</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="userLabel">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>69</y>
        <width>71</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>用户名</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="fakioLabel">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>11</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Fakio 服务器</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabAbout">
     <attribute name="title">
      <string>关于</string>
     </attribute>
     <widget class="QTextBrowser" name="aboutBrowser">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>2</y>
        <width>434</width>
        <height>160</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="StartButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>启动</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>210</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>保存设置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>210</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>取消设置</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Fakio.qrc"/>
 </resources>
 <connections/>
</ui>
